### ISSUE TITLE
Accesses: (Re-)Add CoID to Accesses endpoint

### ISSUE THREAD

**Issue by MartinChristenssonOU** [on-api/on-api#98]
In the original address specification from PI-API we had a mandatory field identifying the CO.
"coID - Unik identifierare för varje KO. "
It was removed in ON-API 2.4

I suggest we bring that field back to support co-operations between several networks or several CO brands in one system.
Otherwise such entities will be force to use population or other fields to distinguish between themselves.

**Comment by junetaw** [comment]
When will it be used and by who?

With the PriceGroup field (that is hopefully added in 2.6) any price information is handled (in combination with PremisesType) and Population will not be used for that purposes. So the CoId will not be relevant to that.

Should it be used in technical issues? Like support? 
In case different "Sub-CO" use the same system but has different customer support organizations then we should also add/alter that endpoint so CoId also gives contact informations. Then it has a purpose which I could support.

Otherwise I do not see a value from a ISP horisont. But I have been wrong before :)
Maybe to CO internally might use it?

**Comment by MartinChristenssonOU** [comment]
I agree that pricing information should be handled with other fields. I see no point in adding it to the API for CO internal use.
Apart from contact information for support, the subCOs or members of a co-operation might very well have separate agreements for service providers. This could very well be handled with the PriceGroup or population field.
All in all, the need for the field seems limited.

**Comment by junetaw** [comment]
This might need to be reopened and reevaluated. 
As the trend is to have more than one CO in a single instance this information might be needed. 
We have several of these combinations now:
- OpenUniverse and GlobalConnect
- Zitius, SSAB and Telia Öppen Fiber
- Njudung and Citynätet 
- etc

So why do we need this:
- We might need to easily see which CO an access belongs to. If a SP is only delivering to a "Sub-CO" and not another, the SP might only want to import accesses it can deliver to.
- We might need to add a get to /invoicespec with CoId as input parameter to be able to get per CoId invoice specifications as the invoices could be sent individually. Today I do not think it is possible to filter be CoId.
- The support information discussed above is needed to be able to make requests to the correct "Sub-Co". Today error reports are probably sent to the "owner" of the api and not to the "Sub-CO" and that means longer lead times and more work.
- The ticketing system might need this so tickets go to the correct department.

There are probably more usages internally in a CO as well. A CO might actually limit the access response if a SP is only delivering to a "Sub-CO" and not all. @MartinChristenssonOU might be able to add to this?

**Comment by MartinChristenssonOU** [comment]
In the address group we decided to keep CO away from the API. I think that you get enough ways to handle this with pricegroups. If an SP only delivers to one brand/"sub-CO", they should only see these addresses anyway, not the ones that they may not sell to. And even if they get all addresses, they should definitely not have any sellable services on them.

**Comment by junetaw** [comment]
Martin makes valid points.
I vote to close this.

**Comment by teemufrosen** [comment]
Working group meeting: Won't do, no need. See above.
